2024-04-17 16:57:33 ERROR Elements not present after 0.5 seconds on page: https://capital.com/it/eur-aud-tasso
2024-04-17 16:57:33 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 304, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x563818f92863 <unknown>
#1 0x563818c888c6 <unknown>
#2 0x563818cd3618 <unknown>
#3 0x563818cd36d1 <unknown>
#4 0x563818d16744 <unknown>
#5 0x563818cf55cd <unknown>
#6 0x563818d13c19 <unknown>
#7 0x563818cf5343 <unknown>
#8 0x563818cc6593 <unknown>
#9 0x563818cc6f5e <unknown>
#10 0x563818f5684b <unknown>
#11 0x563818f5a7a5 <unknown>
#12 0x563818f44571 <unknown>
#13 0x563818f5b332 <unknown>
#14 0x563818f2987f <unknown>
#15 0x563818f81728 <unknown>
#16 0x563818f818fb <unknown>
#17 0x563818f919b4 <unknown>
#18 0x7f1909e94ac3 <unknown>